patches/chrome-browser-flag_descriptions.cc.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/chrome/browser/flag_descriptions.cc b/chrome/browser/flag_descriptions.cc
+index d40ce5d934eed97ed9158a7828faf5dab4324df1..4b9e3755b3f82c8010371e2ca571d2641b84aaec 100644
+--- a/chrome/browser/flag_descriptions.cc
++++ b/chrome/browser/flag_descriptions.cc
+@@ -4431,6 +4431,11 @@ const char kV8VmFutureDescription[] =
+     "This enables upcoming and experimental V8 VM features. "
+     "This flag does not enable experimental JavaScript features.";
+ 
++const char kWebAssemblyJitlessName[] = "WebAssembly jitless mode";
```

Comment:
these would normally be done as chromium_src overrides, but I don't think we need any of these patch changes at all in chrome/browser code. See comment below in render process host impl